controller/workitem_blackbox_test.go

```diff
@@ -3392,3 +3381,25 @@ func (s *WorkItem2Suite) TestCreateAndUpdateWorkItemForEveryWIT() {
 		})
 	}
 }
+
+func (s *WorkItem2Suite) TestDeleteWorkitem() {
+	s.T().Run("ok", func(t *testing.T) {
+		fxt := tf.NewTestFixture(s.T(), s.DB, tf.WorkItems(1))
+		s.svc = testsupport.ServiceAsUser("TestUpdateWI2-Service", *fxt.Identities[0])
+		test.DeleteWorkitemOK(s.T(), s.svc.Context, s.svc, s.workitemCtrl, fxt.WorkItems[0].ID)
+	})
+	s.T().Run("unauthorized", func(t *testing.T) {
+		fxt := tf.NewTestFixture(s.T(), s.DB, tf.WorkItems(1), tf.Identities(2))
```

Comment:
The second identity seems unused.

Reply:
removed unused identity here 1b07bad

Reply:
`tf.WorkItems(1), tf.Identities(1)` is the same a `tf.WorkItems(1)` because a work item requires an identity on its own.

Reply:
@kwk yes. Thanks for pointing that out. removed it here -> 77b6e4e